WhisperAPI/WhisperAPI/Controllers/SelectSuggestionController.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using WhisperAPI.Models;
+using WhisperAPI.Services.Context;
+using WhisperAPI.Services.SelectSuggestion;
+
+namespace WhisperAPI.Controllers
+{
+    [Route("/Whisper/[Controller]")]
+    public class SelectSuggestionController : ContextController
+    {
+        private static readonly log4net.ILog Log = log4net.LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
+
+        private readonly ISelectSuggestionService _selectSuggestionService;
+
+        public SelectSuggestionController(ISelectSuggestionService selectSuggestionService, IContexts contexts)
+            : base(contexts)
+        {
+            this._selectSuggestionService = selectSuggestionService;
+        }
+
+        [HttpPost]
```

Comment:
Move this endpoint to SuggestionsController.cs, change the attribute to [HttpPost("/click")], this way, we don't need an additional controller for that. Plus, it is related to suggestions. Finally, this endpoint should only receive a Guid, why does it receive a SearchQueryy ? This doesn't seem like the right class to use.

Receiving a Guid would also have the advantage to not duplicate the data so we always stay coherent, don't forget that anyone can modify the call made to the API, so we couldn't trust the data in the post body anyway.

You can choose whether you want the Guid to be in the Url or the body, I'd say in the body because Guids are ugly.

The endpoint would look like:
[HttpPost("/click")]
        public IActionResult SelectSuggestion([FromBody] Guid suggestionGuid)